OpenSDS_Volume_Actions_Design.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Status: Pending
+
+Version: Beta
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+Extends the size of a volume. 
+
+## Motivation
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Goals
+
+What will be addressed by the proposal?
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Non-Goals
+
+What will not be addressed by the proposal?
+"Shrinks the size of a volume" will not be addressed.
+
+## Proposal
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
```

Comment:
Are you adding any code to handle in-use volume or are you going to test in-use volume?  If not, please add a note saying "Extend in-use volume will be addressed later".

Reply:
It does not matter whether or not the volume is being used. It is up to the drivers to determine whether volume can extend.